x-pack/test/functional/apps/dashboard/group1/created_by.ts

```diff
@@ -17,7 +17,7 @@ export default function ({ getPageObjects, getService }: FtrProviderContext) {
   const securityService = getService('security');
   const testSubjects = getService('testSubjects');
 
-  describe('created_by', function () {
+  describe.only('created_by', function () {
```

Comment:
remove `.only`

Reply:
Fixed in [34d30e0](https://github.com/elastic/kibana/pull/225522/commits/34d30e0b07e950d88748344606d11d3c8f671205)